.gitignore

```diff
@@ -1 +1,2 @@
 /.idea/inspectionProfiles/
+/.idea/
```

Comment:
Isn't that redundant or am I missing something here?

Reply:
yep - second one is broader. Removing first one.